cmd/bcpd/app/app.go

```diff
@@ -59,6 +59,7 @@ func Router(authFn x.Authenticator, issuer weave.Address) app.Router {
 	// TODO: move to cash upon refactor
 	escrow.RegisterRoutes(r, authFn, namecoin.NewController())
 	multisig.RegisterRoutes(r, authFn)
+	validators.RegisterRoutes(r, authFn, validators.NewController(validators.NewBucket()))
```

Comment:
Same advice I gave Lehajam... unless you really need to, better to make NewContoller take no args, and just initiate the bucket there.

The only reason I did it differently in cash is that I wanted to allow people to slip in different implementations (eg. namecoin) while retaining the same handlers. It was an attempt at reuse, maybe not the cleanest, and definitely only adds confusion if not "sub-classing" of the module is forseen.

Reply:
Yep, just didn't want to change any tests as this passed review and such, so figured why not "plug and play"

Reply:
I'll fix that up shortly

Reply:
Yep, somehow this part got missing from my original impl, so I'm going to try to figure out all the missing bits and pieces and fix em up